tweets/views.py

```diff
@@ -41,4 +41,13 @@ def destroy(self, request, id):
     tweet = get_object_or_404(Tweet, id=id, user_id=request.user.id)
     tweet.delete()
     messages.success(request, "Tweet deleted")
+    return redirect('home')
+
+class RetweetView(View):
+  def post(self, request, original_tweet_id):
+    tweet = get_object_or_404(Tweet, id=original_tweet_id)
+    if request.user.id==tweet.user_id:
+      messages.error(request,'Cannot Retweet self Tweet')
+    Tweet.objects.create(user = request.user, original_tweet = tweet)
+    messages.success(request, "Tweet Retweeted")
     return redirect('home')
```

Comment:
This belongs to `retweets/views.py`

Reply:
Should I manually create a directory inside tweet app and load it on urls page. As because when we create an app the views.py is by default there as a blank page.

Reply:
I don't think that creating seperate app for retweet is good idea.

Reply:
Done please check now.